ReactWindows/ReactNative.Net46/Views/Web/ReactWebViewManager.cs

```diff
@@ -150,8 +149,7 @@ public override void ReceiveCommand(WebBrowser view, int commandId, JArray args)
                     view.Refresh();
                     break;
                 default:
-                    throw new InvalidOperationException(
-                        Invariant($"Unsupported command '{commandId}' received by '{typeof(ReactWebViewManager)}'."));
```

Comment:
Can't we just mock the System.FormattableString static class for .NET 4.5.1 so we don't have to eliminate the `using static` and the `Invariant` calls?  We can just make the `Invariant` calls an identity function on .NET 4.5.1....